### DISCUSSION TITLE
Can we set global configs to an engine for a specific user/group?

### DISCUSSION THREAD

**Discussion by gabry-lab** [apache/kyuubi#2454] [Q&A] [answered]
Hi, folks,
sometimes, we want to set a global & readonly configs to an engine for a specific user and group. So, my question is that, does Kyuubi support this feature?

**Comment by yaooqinn** [comment, marked as answer]
we have kyuubi.session.conf.ignore.list/kyuubi.session.conf.restrict.list for banning configurations from client side, but not for particular user or group.

You may need the developer API - `org.apache.kyuubi.plugin.SessionConfAdvisor` to implement such a rule.
